src/clients/firewall_auto.rb

```diff
@@ -87,9 +87,7 @@ def main
         SuSEFirewall.SetStartService(SuSEFirewall.GetEnableService)
       # Import configuration
       elsif @func == "Import"
-        @ret = SuSEFirewall.Import(
-          Convert.convert(@param, :from => "map", :to => "map <string, any>")
-        )
+        @ret = SuSEFirewall.read_and_import(@param)
```

Comment:
then calling import via client and via API will be different which is confusing. I think bug was original behavior so it should be changed.